src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+  const adjustedTabId = selectedTab.id;
```

Comment:
you already have selectedTabId with default value, it means you don't need additional variables or smth 
just use `selectedTabId` from props